### PR TITLE
Extended count of custom fields to 120

### PR DESCRIPTION
**Pull request by sevcikmartin**

### CODE REVIEW

File: src/store/keys/OpportunityContextKeys.ts [hunk 1]
```diff
@@ -163,5 +163,27 @@ export enum OpportunityContextKeys {
     CUSTOM_FIELD_98 = 'opp.csf98',
     CUSTOM_FIELD_99 = 'opp.csf99',
 
-    CUSTOM_FIELD_100 = 'opp.csf100'
+    CUSTOM_FIELD_100 = 'opp.csf100',
```
Comment (nimal-outreach) [inline src/store/keys/OpportunityContextKeys.ts:166]: Why do you think opportunity has more the 100 custom fields?

According to API it has exactly 100?
https://api.outreach.io/api/v2/docs#opportunity

File: src/store/keys/AccountContextKeys.ts [hunk 2]
```diff
@@ -147,5 +147,27 @@ export enum AccountContextKeys {
     CUSTOM_FIELD_98 = 'acc.csf98',
     CUSTOM_FIELD_99 = 'acc.csf99',
 
-    CUSTOM_FIELD_100 = 'acc.csf100'
+    CUSTOM_FIELD_100 = 'acc.csf100',
```
Comment (nimal-outreach) [inline src/store/keys/AccountContextKeys.ts:150]: Same question for account - according to APi spec it has 100 custom fields not 120?

https://api.outreach.io/api/v2/docs#account

**Review by nimal-outreach — CHANGES_REQUESTED** [review]
Any change in SDK in order to result with NPM module you can use in flagship needs to include incrementing version in package.json

File: src/store/keys/OpportunityContextKeys.ts (hunk 1 above)
Comment (sevcikmartin) [reply src/store/keys/OpportunityContextKeys.ts:166]: This custom fields exceeding for Account and Opportunity was mentioned last week on the All-hands-meeting, Martin Petrla have worked on that.

**Pull request closed without merging**